include/deal.II/base/bounding_box.h

```diff
@@ -193,6 +267,61 @@ class BoundingBox
   std::pair<Point<spacedim, Number>, Point<spacedim, Number>> boundary_points;
 };
 
+
+/**
+ * Returns the unit box [0,1]^dim.
+ */
+template <int dim, typename Number = double>
+BoundingBox<dim, Number>
+create_unit_box();
```

Comment:
May I ask to just rename this function to `create_unit_bounding_box()` to make its name conform to the classname? You might always want to put `@relates BoundingBox` into the documentation so that it shows up on the relevant doxygen page.